src/src/rx_main.cpp

```diff
@@ -254,7 +254,7 @@ void SetRFLinkRate(uint8_t index) // Set speed of RF link
     bool invertIQ = UID[5] & 0x01;
 
     hwTimer.updateInterval(ModParams->interval);
-    Radio.Config(ModParams->bw, ModParams->sf, ModParams->cr, GetInitialFreq(), ModParams->PreambleLen, invertIQ, ModParams->PayloadLength);
+    Radio.Config(ModParams->bw, ModParams->sf, ModParams->cr, GetInitialFreq(), ModParams->PreambleLen, invertIQ, ModParams->PayloadLength, 0);
```

Comment:
why timeout is not set for the RX? I think it could be beneficial for the rx as well to make it more robust. What do you think?

Reply:
The RX operates in continuous mode so there is no timeout. When disconnected it doesn't know when to start listening for a packet to line up reception. Once it is synced we could theoretically add a timeout, but having it receive switch modes once synced seems like extra work that might not have much benefit (not starting a receive during our silent period).